tests/qit/security.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+cwd=$(pwd)
+WCP_ROOT=$cwd
+QIT_ROOT="$cwd/tests/qit"
+EXTENSION_NAME="woocommerce-payments"
+
+
+#Load local env variables if present.
+if [[ -f "$QIT_ROOT/config/local.env" ]]; then
+	. "$QIT_ROOT/config/local.env"
+fi
+
+# Check if QIT_USER and QIT_APP_PASSWORD are set and not empty
+if [[ -z $QIT_USER ]] || [[ -z $QIT_PASSWORD ]] ; then
+    echo "QIT_USER or QIT_APP_PASSWORD environment variables are not set or empty. Please set them before running the script."
```

Comment:
Maybe mention to "set them in a local env file" to improve the instruction a little?

Reply:
Done.